### PR TITLE
tbls: replace verifier with pubshares

### PR DESCRIPTION
**Pull request by dB2510**
Refactors tbls to use pubshares from start and remove verifier from the workflow.

category: refactor
ticket: #596

### CODE REVIEW

**Review by corverroos — APPROVED** [review]
🚀

File: cluster/testdata/TestEncode_lock_json.golden [hunk 1]
```diff
@@ -31,20 +31,12 @@
  },
  "distributed_validators": [
   {
-   "distributed_public_key": "0x7b182e046410f44bc4b0f3f03a0d06820a30f257",
-   "threshold_verifiers": [
```
Comment (xenowits) [inline cluster/testdata/TestEncode_lock_json.golden:35]: shouldn't `threshold_verifiers` be replaced with `pubshares`?

File: dkg/frost.go [hunk 2]
```diff
@@ -219,7 +219,7 @@ func makeShares(
 	targetID uint32,
 ) ([]share, error) {
 	// Get set of public shares for each validator.
-	pubShares := make(map[uint32]map[uint32]*bls_sig.PublicKey) // map[ValIdx]map[SourceID]*bls_sig.PublicKey
+	pubShares := make(map[uint32]map[int]*bls_sig.PublicKey) // map[ValIdx]map[SourceID]*bls_sig.PublicKey
```
Comment (xenowits) [inline dkg/frost.go:222]: nit: why `int` instead of `uint32`?

File: dkg/keycast.go [hunk 3]
```diff
@@ -185,9 +175,9 @@ func createShares(numValidators, numNodes, threshold int, random io.Reader) ([][
 
 		for ni := 0; ni < numNodes; ni++ {
 			resp[ni] = append(resp[ni], share{
-				PubKey:      pubkey,
-				Verifier:    verifier,
-				SecretShare: shares[ni],
+				PubKey:       tss.PublicKey(),
+				PublicShares: tss.PublicShares(),
+				SecretShare:  shares[ni],
```
Comment (xenowits) [inline dkg/keycast.go:180]: nit: can we rename the loop variable `ni`?

File: dkg/frost.go (hunk 2 above)
Comment (dB2510) [reply dkg/frost.go:222]: while going through the code i didn't found any specific reason to use uint32 for dkg, that's why i went with fundamental types such as int

can change if there was any specific reason behind the usage of uint32 for dkg purposes

**Pull request merged**